tests/probes_test.go

```diff
@@ -47,11 +47,15 @@ var _ = Describe("[ref_id:1182]Probes", func() {
 			},
 		}
 		table.DescribeTable("should succeed", func(readinessProbe *v12.Probe, serverStarter func(vmi *v12.VirtualMachineInstance, port int)) {
+			if tests.IsRunningOnKindInfra() && readinessProbe == httpProbe {
+				Skip("Skip http probe test till PR https://github.com/kubevirt/kubevirt/pull/3224 is merged")
```

Comment:
You could probably refer this [issue](https://github.com/kubevirt/kubevirt/issues/3247) I've opened about it instead; I'll abandon that PR soon, in favor of another solution.

Reply:
done